js/everything.js

```diff
@@ -651,6 +658,15 @@ $(function() {
         $("#demo").hide();
     }
 
+    $("input[name='toggleLoop']").change(function() {
+        if ($(this).is(":checked")) {
```

Comment:
This if/else logic should be simplified to `ZenPlayer.isLoop = $(this).is(":checked");`